.ci/script/build.sh

```diff
@@ -28,6 +28,16 @@ rbenv versions
 rbenv global $JRUBY_VERSION
 ruby --version # just to double-check...
 which -a ruby
+ruby_bin_file=$(which ruby)
+export BUNDLE_CMD="$ruby_bin_file -S bundle"
```

Comment:
@tarekziade is there a better way to invoke this logic from https://github.com/elastic/ent-search-ci-build/blob/main/lib/entsearchci/generator/ruby.rb without having to use `perform-ci-build`?